kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -1893,6 +1902,33 @@ def test_duplicate_submission_as_anotheruser_with_partial_perms(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self._check_duplicate(response, submission)
 
+    def test_duplicate_submission_with_extras(self):
+        dummy_extra = {
+            'q1': {
+                'transcript': {
+                    'value': 'dummy transcription',
+                    'languageCode': 'en',
+                },
+                'translation': {
+                    'tx1': {
+                        'value': 'dummy translation',
+                        'languageCode': 'xx',
+                    }
+                },
+            },
+            'submission': self.submission['_uuid']
+        }
+        self.asset.update_submission_extra(dummy_extra)
+        response = self.client.post(self.submission_url, {'format': 'json'})
+        duplicated_submission = response.data
+        duplicated_extra = self.asset.submission_extras.filter(
+            submission_uuid=duplicated_submission['_uuid']
+        ).first()
+        assert duplicated_extra.content['q1']['translation']['tx1']['value'] == \
+            dummy_extra['q1']['translation']['tx1']['value']
+        assert duplicated_extra.content['q1']['transcript']['value'] == \
+            dummy_extra['q1']['transcript']['value']
```

Comment:
Let's use black to format long lines. 
Something like that 

```python 
        assert (
            duplicated_extra.content['q1']['translation']['tx1']['value']
            == dummy_extra['q1']['translation']['tx1']['value']
        )
        assert (
            duplicated_extra.content['q1']['transcript']['value']
            == dummy_extra['q1']['transcript']['value']
        )
```

Reply:
I thought black reformats the whole file always, but I just saw that I can specify a range of lines